backend/de.metas.material/dispo-service/src/main/sql/postgresql/system/75-material-dispo/5757390_sys_gh23733_MaterialDispo_To_Forecast_And_ManufacturingCandidate.sql

```diff
@@ -0,0 +1,174 @@
+-- Run mode: SWING_CLIENT
+
+-- Name: M_Forecast target for MD_Candidate
+-- 2025-06-11T09:54:00.281Z
+INSERT INTO AD_Reference (AD_Client_ID,AD_Org_ID,AD_Reference_ID,Created,CreatedBy,EntityType,IsActive,IsOrderByValue,Name,Updated,UpdatedBy,ValidationType) VALUES (0,0,541957,TO_TIMESTAMP('2025-06-11 09:54:00.032000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'de.metas.material.dispo','Y','N','M_Forecast target for MD_Candidate',TO_TIMESTAMP('2025-06-11 09:54:00.032000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'T')
+;
+
+-- 2025-06-11T09:54:00.291Z
+INSERT INTO AD_Reference_Trl (AD_Language,AD_Reference_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Reference_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Reference t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Reference_ID=541957 AND NOT EXISTS (SELECT 1 FROM AD_Reference_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Reference_ID=t.AD_Reference_ID)
+;
+
+-- Reference: M_Forecast target for MD_Candidate
+-- Table: M_Forecast
+-- Key: M_Forecast.M_Forecast_ID
+-- 2025-06-11T09:56:03.681Z
+INSERT INTO AD_Ref_Table (AD_Client_ID,AD_Key,AD_Org_ID,AD_Reference_ID,AD_Table_ID,AD_Window_ID,Created,CreatedBy,EntityType,IsActive,IsValueDisplayed,ShowInactiveValues,Updated,UpdatedBy,WhereClause) VALUES (0,11918,0,541957,720,328,TO_TIMESTAMP('2025-06-11 09:56:03.642000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'de.metas.material.dispo','Y','N','N',TO_TIMESTAMP('2025-06-11 09:56:03.642000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'EXISTS(SELECT 1 from m_forecast forecast where forecast.m_forecast_id = @M_Forecast_ID/-1@ AND forecast.m_forecast_id = m_forecast.m_forecast_id)')
```

Comment:
not sure, but do we really need this `exists` stuff?
why not just having `m_forecast.m_forecast_id = @M_Forecast_ID/-1@` as the whereclause